Docker merge to main (not dev) branch

Harry-Yu-Shuhang opened the pull request:
See [#232 ](https://github.com/Open-LLM-VTuber/Open-LLM-VTuber/pull/232)

Harry-Yu-Shuhang committed:
Update dockerfile to tim's current version

Harry-Yu-Shuhang committed:
Create docker-compose.yml

Harry-Yu-Shuhang committed:
debug uv sync

Harry-Yu-Shuhang committed:
Create build_docker.sub

Harry-Yu-Shuhang committed:
slurm compose

Harry-Yu-Shuhang committed:
Add Tsinghua Source

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Add git and dockerignore

Harry-Yu-Shuhang committed:
Divide into CPU and GPU

ylxmf2005 committed:
删除工作流中的 AWS/R2 相关代码

ylxmf2005 committed:
Merge branch 'dev' of https://github.com/Open-LLM-VTuber/Open-LLM-VTuber into dev

ylxmf2005 committed:
Revert "删除工作流中的 AWS/R2 相关代码"

This reverts commit 8e7e58a6f68d6dd235867b8fcce3f405b4ccd1e3.

ylxmf2005 committed:
update frontend submodule

ylxmf2005 committed:
bump version to 1.2.1

ylxmf2005 committed:
add _cleanup_failed_connection

ylxmf2005 committed:
Create CLAUDE.md

ylxmf2005 committed:
update frontend

ylxmf2005 committed:
ruff format

ylxmf2005 committed:
Merge branch 'main' into dev

ylxmf2005 committed:
update frontend

Harry-Yu-Shuhang committed:
Delete submodule pull

Harry-Yu-Shuhang committed:
Merge branch 'main' into docker

Harry-Yu-Shuhang committed:
Merge branch 'dev' into docker
Update to dev

Harry-Yu-Shuhang committed:
cpu docker OK

Harry-Yu-Shuhang committed:
Merge branch 'Open-LLM-VTuber:main' into docker

Harry-Yu-Shuhang committed:
cpu Done

Harry-Yu-Shuhang committed:
改名

Harry-Yu-Shuhang committed:
ollama docker

Harry-Yu-Shuhang committed:
Docker加入ollama

用法：
打包image：docker compose -f ./docker-compose.yml build
用户使用：把conf.yaml挂载到/app/conf
以及填一个12393
具体后续详细说明

Harry-Yu-Shuhang committed:
注释改英文

Harry-Yu-Shuhang committed:
CPU Support

Harry-Yu-Shuhang committed:
Update docker-compose.yml

Harry-Yu-Shuhang committed:
Add Tsinghua Source

Harry-Yu-Shuhang committed:
Custom live2d-models and characters

Harry-Yu-Shuhang committed:
Merge branch 'Open-LLM-VTuber:main' into docker

Harry-Yu-Shuhang committed:
Update README_DOCKER.md

Harry-Yu-Shuhang committed:
Merge branch 'docker' of https://github.com/Harry-Yu-Shuhang/Open-LLM-VTuber into docker

Harry-Yu-Shuhang committed:
Update README_DOCKER.md

Harry-Yu-Shuhang committed:
Update Readme

Harry-Yu-Shuhang committed:
Merge branch 'Open-LLM-VTuber:main' into docker

Harry-Yu-Shuhang committed:
chore: update requirements files (bot)

Harry-Yu-Shuhang committed:
single line cmds to support powershell

Harry-Yu-Shuhang committed:
Merge branch 'docker' of https://github.com/Harry-Yu-Shuhang/Open-LLM-VTuber into docker

Harry-Yu-Shuhang committed:
Debug

Harry-Yu-Shuhang committed:
Update README_DOCKER_CN.md

Harry-Yu-Shuhang committed:
Update README_DOCKER_EN.md

Harry-Yu-Shuhang committed:
Merge branch 'Open-LLM-VTuber:main' into docker

Harry-Yu-Shuhang committed:
chore: update requirements files (bot)

Harry-Yu-Shuhang committed:
docker添加live2d和character

Harry-Yu-Shuhang committed:
add deleta docker data

Harry-Yu-Shuhang committed:
Update README_DOCKER_CN.md

Harry-Yu-Shuhang committed:
Add server config

Harry-Yu-Shuhang committed:
Add server preparation

Harry-Yu-Shuhang committed:
Add avatar

Harry-Yu-Shuhang committed:
禁止用户挂载models

Harry-Yu-Shuhang committed:
Update server_config_cn.sh

Harry-Yu-Shuhang committed:
Update requirements.txt

Harry-Yu-Shuhang committed:
Update dockerfile

Delete unnecessary comments

Harry-Yu-Shuhang committed:
docker support self-defined ollama model

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
Update dockerfile

Harry-Yu-Shuhang committed:
dont prepare for ollama when not ollama

Harry-Yu-Shuhang committed:
Add Ollama pull

Harry-Yu-Shuhang committed:
ollama not storaged in docker

On user's pc is more reasonable

Harry-Yu-Shuhang committed:
docker回滚

Harry-Yu-Shuhang committed:
Default model: qwen2.5:latest

Harry-Yu-Shuhang committed:
docker keep same logic as readme

but expose files that user can self-define in volumes

Harry-Yu-Shuhang committed:
update docker compose

Harry-Yu-Shuhang committed:
update docker readme

Harry-Yu-Shuhang committed:
Update docker-compose.yml

Harry-Yu-Shuhang committed:
create docker auto update

Harry-Yu-Shuhang committed:
Docker comment translate to English

Harry-Yu-Shuhang committed:
docker yml change to latest

Harry-Yu-Shuhang committed:
Delete docker-compose.yml

Harry-Yu-Shuhang committed:
Update README_DOCKER

t41372 reviewed (changes requested):
我没有测试过这个 docker 镜像的构建和构建之后到底能不能跑，我就先相信你啦

olv2 之后，我们得构建一些自动测试流程才行... 我感觉我似乎一直在说类似 "olv2 之后我们要xxx" 之类的话...

处理一下我写的评论。我先去看看 blacksmith 做 ci/cd 了。到时候我会直接把相关的东西推到这个 pr 的分支来，然后我们就在这里讨论/测试。

另外，这个 pr 我会用 squash 之后再合并的方式。olv2 之后的 pr 大多也会用 squash 再合并的方式，不然我们主仓库的 git 历史记录可读性会非常糟糕。

t41372 committed:
ci(docker): add docker build action based on blacksmith

t41372 committed:
ci(docker): allow docker ci to run on pr branch

t41372 committed:
fix(ci): fix docker ci

Harry-Yu-Shuhang committed:
Update README_DOCKER_EN.md

Harry-Yu-Shuhang committed:
Merge branch 'docker' of ssh://ssh.github.com:443/Harry-Yu-Shuhang/Open-LLM-VTuber into docker

Harry-Yu-Shuhang commented on .dockerignore:
```diff
@@ -0,0 +1,79 @@
+!live2d-models/shizuku/**
+
+# Characters
+characters/*
```
测试过。你可以看一下现在那个OLV-Docker-Config的仓库，里头把所有的用户文件都提取出来了，包括这个characters文件夹，容器里头试了，也可以选人设

Harry-Yu-Shuhang committed:
Merge branch 'Open-LLM-VTuber:main' into docker

t41372 committed:
chore: remove dockerfile dir & scripts

removed dockerfile folder and other things in accordance with our discussion in the pr, now that the new action runner based on blacksmith allowed us to build the image with ci

t41372 committed:
fix: broken ci due to wrong dockerfile location

t41372 commented:
我把 dockerfile 目录删掉了，把你里面写的文档改了改丢到文档站去了，只保留了一个 dockerfile 文件。
如果你觉得没问题，我就合并了。

Harry-Yu-Shuhang commented on dockerfiles/change_default_conf_to_docker.py:
其实就是把主仓库配置文件里的127.0.0.1改成host.docker.internal,我自己用的。删了也可以，我本地放着就行

t41372 reviewed (approved):
嗯，code rabbit 没玩明白。之后找 ethan 救救。我先合并这个 pr 了。
另外，和之前一样，我会先 squash 之后再合并。
